avm-transpiler/src/transpile.rs

```diff
@@ -989,32 +989,6 @@ fn handle_black_box_function(avm_instrs: &mut Vec<AvmInstruction>, operation: &B
                 ..Default::default()
             });
         }
-        BlackBoxOp::PedersenHash { inputs, domain_separator, output } => {
```

Comment:
Beware that I think Tom has a few PRs doing similar things

Reply:
I'm blocked by his PR, so maybe i'll rebase on his